native/sqlc.c

```diff
@@ -6,32 +6,36 @@
 
 #include "sqlite3.h"
 
-#define BASE_HANDLE_OFFSET 0x100000000LL
-
 #ifdef SQLC_KEEP_ANDROID_LOG
 // ref: http://www.ibm.com/developerworks/opensource/tutorials/os-androidndk/index.html
 #define MYLOG(...) __android_log_print(ANDROID_LOG_VERBOSE, "sqlc", __VA_ARGS__)
 #else
 #define MYLOG(...) ;
 #endif
 
-#define HANDLE_FROM_VP(p) ( BASE_HANDLE_OFFSET + ( (unsigned char *)(p) - (unsigned char *)NULL ) )
-#define HANDLE_TO_VP(h) (void *)( (unsigned char *)NULL + (ptrdiff_t)((h) - BASE_HANDLE_OFFSET) )
+#define HANDLE_FROM_VP(p) (( (unsigned char *)(p) - (unsigned char *)NULL ) )
+#define HANDLE_TO_VP(h) (void *)( (unsigned char *)NULL + (ptrdiff_t)((h)) )
```

Comment:
I guess this no longer is required. Or is there something to it ?

Reply:
I removed the BASE_OFFSET and left it there to be an helper for the conversions `pointer -> long` and `long -> pointer`, and to minimize the changes in the code as well.